custom_components/kia_uvo/sensor.py

```diff
@@ -78,6 +78,20 @@
         native_unit_of_measurement=PERCENTAGE,
         device_class=SensorDeviceClass.BATTERY,
     ),
+    SensorEntityDescription(
+        key="ev_battery_soh_percentage",
+        name="EV State of Health Battery",
+        native_unit_of_measurement=PERCENTAGE,
+        device_class=SensorDeviceClass.BATTERY,
+    ),
+    SensorEntityDescription(
+        key="ev_battery_remain",
+        name="EV Battery Level",
```

Comment:
how is this different than `ev_battery_percentage`? (line 75)

Reply:
It's a number value and I am hoping for a higher precision. My car is showing the following:
ev_battery_remain: 191700
ev_battery_capacity: 359280
ev_battery_percentage: 57.5
If I calculate then 191700 / 3592.8 = 53.35. So it seem like an interesting discrepancy. I do not know yet if this numbers are perhaps not linear and what kind of unit they are.

Reply:
![BatteryValues](https://github.com/Hyundai-Kia-Connect/kia_uvo/assets/12473381/feffa694-cc55-4383-9eb5-271e69ce717a)

Reply:
Are they helpful to expose until we know what they mean?  Do they show up in the native app anywhere?

Reply:
No they do not show up. I can only see the battery percentage in steps of .5 percentage which I think is not very precise. So I am hoping for more information. 
The SoH is also not visible in the app but I think it is a very important information.

Reply:
Alright we can merge without units for now.  Maybe we will learn more with time.